pnpm-workspace.yaml

```diff
@@ -1,5 +1,5 @@
 packages:
-  - "azure/packages/**"
-  - "examples/**"
-  - "experimental/**"
-  - "packages/**"
+    - "azure/packages/**"
```

Comment:
Is this a setting we can change? It seems unnecessary to increase the indentation here.

Reply:
It appears our policy is 4 space indent for yaml (since apparently using tabs in it is bad). This now complies with it. If someone wants to change our formatting settings for yaml, that seems like a separate issue.